--- asciidoc-9.1.0/asciidoc.py~	2021-02-11 01:56:34.000000000 +0000
+++ asciidoc-9.1.0/asciidoc.py	2021-06-02 18:01:05.503837199 +0000
@@ -6448,8 +6448,7 @@
                                     'section-numbers', 'verbose', 'version', 'safe', 'unsafe',
                                     'doctest', 'filter=', 'theme='])
     except getopt.GetoptError:
-        message.stderr('illegal command options')
-        sys.exit(1)
+        None
     opt_names = [opt[0] for opt in opts]
     if '--doctest' in opt_names:
         # Run module doctests.
